src/lib/module.ts

```diff
@@ -35,6 +36,10 @@ export type ModuleConfig = {
   env?: Record<string, string>;
   /** Custom Labels for Kubernetes Objects */
   customLabels?: CustomLabels;
+  /** Custom RBAC rules */
+  rbac?: PolicyRule[];
+  /** The RBAC mode; if "scoped", generates scoped rules, otherwise uses wildcard rules. */
+  rbacMode?: string;
```

Comment:
This is out of scope for this issue, better scoped for a new issue

Reply:
Going to go ahead and prioritize issue https://github.com/defenseunicorns/pepr/issues/1327 so we can get this merged.